tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -2422,31 +2436,44 @@ tensorflow::Status ConvertStridedSlice(OpConverterParams* params) {
           attr, " is not supported for StridedSlice, at ", node_def.name());
     }
   }
+  const int begin_mask = attrs.get<int>("begin_mask");
+  const int end_mask = attrs.get<int>("end_mask");
+  // Check that batch dimension is unmodified. If the batch size is -1 and the
+  // end mask is not set, we do not convert the op since the batch dim could
+  // potentially be modified.
+  if ((!(begin_mask & 1) && begin[0] != 0) || stride[0] != 1 ||
```

Comment:
Also fix this.

Reply:
This bool expression is too long, please split it into smaller pieces and add comments for readability. It'll also help if you can split the one for ConvertSlice.